socketServer.cpp

```diff
@@ -101,7 +101,8 @@ void SocketServer::processBinaryMessage(QByteArray message)
                     int lle = json.contains("lle") && json.value("lle").toString() == "Yes";
                     writeLog(QString("creating ") + (lle ? "LLE" : "HLE") + " room", json.value("room_name").toString(), json.value("game_name").toString(), port);
 
-                    ServerThread *serverThread = new ServerThread(port, this);
+                    bool useClientCount = json.contains("use_client_count") ? json.value("use_client_count").toBool() : false;
```

Comment:
I have a corresponding change to the mupen64plus-gui that sets this variable based on an option in the Create Room form.